Source/WebCore/css/CSSFontFeatureValue.cpp

```diff
@@ -46,7 +46,7 @@ String CSSFontFeatureValue::customCSSText() const
         builder.append(c);
     builder.append('"');
     // Omit the value if it's `1` as `1` is implied by default.
-    if (m_value->isCalculated() || m_value->resolveAsIntegerNoConversionDataRequired() != 1)
+    if (auto integerValue = m_value->resolveAsIntegerIfNotCalculated(); !integerValue || *integerValue != 1)
```

Comment:
Am I remembering correctly that `std::optional` makes `integerValue != 1` do the same thing as `!integerValue || *integerValue != 1`?

Reply:
You are! I had no idea that was a thing! That's very nice.